designs/administrator/future-openshift/log toolbars/index.md

```diff
@@ -0,0 +1,51 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Log toolbar updates
+
+Outcomes:
+* A new log viewer toolbar design using native PatternFly components and appropriate wrapping/collapsing behavior for various screen sizes
+* Support for Wrap lines and Toggle timestamps
+* Support for the Download All button that appears in the Pipelines log viewer
+* Support for at least 1 `ConsoleExternalLogLink` resource that the user may have added
+* Incorporate a find or search feature 
+* Design for the possibility of allowing users to view more than 1000 lines of logs in the future
+
```

Comment:
Hey @LPutney I was asked to give this PR a review, and I know a lot of hard work has already gone in. I thought I would share that coming in as a designer from DevTools i don't understand the context or background reading these bullets, so maybe if you add in some supporting info about why these changes were added (may just need to be 1-2 sentences) then that would help the reader understand the doc. I don't think this comment needs to hold you up, just thought I would share my view here.